cpu/cc2538/include/periph_cpu.h

```diff
@@ -241,6 +241,71 @@ typedef struct {
     uint_fast8_t cfg;       /**< timer config word */
 } timer_conf_t;
 
+/**
+ * @name   Override resolution options
+ * @{
+ */
+#define HAVE_ADC_RES_T
+typedef enum {
+    ADC_RES_6BIT  =             (0xa00),    /**< not supported by hardware */
```

Comment:
@A-Paul this defines are used in similar fashion throught RIOTs ADC low-level drivers. I don't know why its done this way, but this here is consistent with other drivers.